### ISSUE TITLE
React exclusion rules don't seem to work

### ISSUE THREAD

**Issue by houshuang** [lydell/eslint-config-prettier#3]
It still gives me formatting errors, even though I thought I'd followed all the instructions:

```
/Users/stian/src/FROG/frog/imports/ui/Activities.jsx
  39:9  error  Missing parentheses around multilines JSX  react/jsx-wrap-multilines
```

Here is my entire .eslintrc file (many of the items should be unnecessary with this config

```
{
	'plugins': ['flowtype', 'prettier'],
	"globals": {
		"window": true
	},
	'rules': {

		'import/extensions': 'off',
		'import/no-extraneous-dependencies': 'off',
		'import/no-unresolved': 'off',
		'no-underscore-dangle': 'off',
		'no-unused-vars': ["error", { "varsIgnorePattern": "_" }],
		'import/prefer-default-export': 'off',
		'prefer-template': 'off',
		'react/prop-types': 'off',
		'react/no-multi-comp': 'off',
		'jsx-a11y/href-no-hash': 'off',
		"react/jsx-filename-extension": 'off',
		'react/jsx-closing-bracket-location': 'off',
		'react/jsx-indent-props': 'off',
		'react/jsx-indent': 'off',
		'no-return-assign': 'off',
		'no-throw-literal': 'off',
		'no-use-before-define': 'off',
		'no-param-reassign': ['error', { 'props': false }],
		"prettier/prettier": ['warn', {'singleQuote': true}],
		'no-nested-ternary': 'off',
		'no-constant-condition': 'off'

	},
	"parserOptions": {
		"ecmaVersion": 2016,
		"sourceType": "module",
		"ecmaFeatures": {
			"jsx": true
		}
	},
	"env": {
		"es6": true,
		"node": true
	},
	'extends': ['airbnb', 'plugin:flowtype/recommended', 'prettier', 'prettier/react' ]
}
```

Thank you!

**Comment by lydell** [comment]
Hi!

I tried to reproduce your problem in this gist: https://gist.github.com/lydell/59a69352139182b9465f01f1d9207587

But I couldn't. :(

1. Clone the gist
2. `npm install`
3. `npm test`

ESLint does not report any errors.

Could you provide more information? The best thing would be if you could modify my gist so it reproduces your problem.

**Comment by lydell** [comment]
There seems like there was a problem with the React rules after all. I've pushed a fix for that, and will make a new release soon.

TODO: Create a test file for React that fails eslint without eslint-config-prettier, and succeeds with it.

**Comment by smably** [comment]
I'm having a similar problem, but with [ternaries in JSX](https://jlongster.github.io/prettier/#%7B%22content%22%3A%22const%20optionalComponent%20%3D%20this.props.visible%5Cn%20%20%3F%20(%3CComponent%5Cn%20%20%20%20prop%3D%7Bthis.props.prop%7D%5Cn%20%20%20%20otherProp%3D%7Bthis.props.otherProp%7D%5Cn%20%20%20%20lastProp%3D%7Bthis.props.lastProp%7D%5Cn%20%20%2F%3E)%5Cn%20%20%3A%20null%3B%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3Afalse%2C%22bracketSpacing%22%3Atrue%2C%22doc%22%3Afalse%7D%7D). `react/jsx-wrap-multilines` and `react/jsx-indent-props` complain if indentation is not like the original.

**Comment by lydell** [comment]
Fixed in version 1.1.0.

There are now tests so that things like this shouldn't happen again.
